src/meltano/cli/cli.py

```diff
@@ -112,7 +112,12 @@ def cli(
     try:
         project = Project.find()
         setup_logging(project)
-        logger.debug("meltano %s, %s", __version__, platform.system())
+        logger.debug(
+            "meltano %s, Python %s, %s",
```

Comment:
We may also want to include `platform.machine()` (e.g. `arm64`)

Reply:
14cf621abcf919b479ca889c9055a756c042942d